modules/whatsInStandard.js

```diff
@@ -0,0 +1,45 @@
+const rp = require("request-promise-native");
+const _ = require("lodash");
+const Discord = require("discord.js");
+
+class WhatsInStandard{
+    constructor() {
+        this.api = "http://whatsinstandard.com/api/4/sets.json";
+        this.commands = ["standard"];
+    }
+
+    getCommands(){
+        return this.commands;
+    }
+
+    generateEmbed(setList){
+        const currentDate = new Date();
+        const removedFutureSetList = setList.filter(set=>{
+            const releaseDate = new Date(set.enter_date);
+            return currentDate.getTime()>=releaseDate.getTime();
+        });
+        const groupedSetList = _.groupBy(removedFutureSetList,'rough_exit_date');
+        let description = "";
+        _.forEach(groupedSetList,(value,key)=>{
```

Comment:
Maybe use native `Array.forEach` instead of lodash here?

Reply:
GroupedSetList is an object with the rotation dates as values containing an array of sets that rotate.
To use `Array.forEach` I would need to use another way than `_.groupBy` to group them by rotation date.

Reply:
Ooops, my bad!